include/vapor/SliceRenderer.h

```diff
@@ -59,35 +60,22 @@ class RENDER_API SliceRenderer : public Renderer {
         glm::vec2 twoD;
     };
 
-    void _renderFast() const;
-
-    void                   _rotate();
-    void                   _findIntercepts(glm::vec3 &origin, glm::vec3 &normal, std::vector<_vertexIn2dAnd3d> &vertices, bool stretch) const;
-    stack<glm::vec2>       _2DConvexHull(std::vector<_vertexIn2dAnd3d> &vertices) const;
-    glm::vec3              _inverseProjection(float x, float y) const;
-    std::vector<glm::vec2> _makeRectangle2D(const std::vector<_vertexIn2dAnd3d> &vertices, stack<glm::vec2> &polygon2D) const;
-    std::vector<glm::vec3> _makeRectangle3D(const std::vector<_vertexIn2dAnd3d> &vertices, stack<glm::vec2> &polygon2D) const;
-    std::vector<glm::vec3> _makePolygon3D(const std::vector<_vertexIn2dAnd3d> &vertices, stack<glm::vec2> &polygon2D) const;
-    void                   _drawDebugPolygons() const;
-
     void _initVAO();
     void _initTexCoordVBO();
     void _initVertexVBO();
 
     bool      _isColormapCacheDirty() const;
     bool      _isDataCacheDirty() const;
     bool      _isBoxCacheDirty() const;
-    void      _getModifiedExtents(vector<double> &min, vector<double> &max) const;
+    void      _getExtents(vector<double> &min, vector<double> &max) const;
     int       _saveCacheParams();
     void      _resetColormapCache();
-    int       _resetBoxCache();
-    int       _resetDataCache();
+    void      _resetCache();
     void      _initTextures();
     void      _createDataTexture(float *dataValues);
-    int       _saveTextureData();
-    void      _populateData(float *dataValues, Grid *grid) const;
-    glm::vec3 _getOrthogonal(const glm::vec3 u) const;
-    glm::vec3 _rotateVector(glm::vec3 vector, glm::quat rotation) const;
+    int       _regenerateSlice();
+    int       _getGrid3D(Grid*& grid) const;
+    void      _drawDebugPolygons();
 
     double _newWaySeconds;
```

Comment:
Some of these private member variables and functions don't appear to be used. Please delete or ifdef out if needed for diagnostics.

Reply:
Fixed.

Reply:
There are still unused private member variables, e.g. _newWaySeconds

Reply:
My bad.  I took another pass and removed all unused private members and functions.